tests/test_launcher.py

```diff
@@ -171,7 +171,6 @@ def test_case_load():
     session.exit()
 
 
-@pytest.mark.standalone
```

Comment:
These tests originally had some issue in running them in github, checking if the issue still exists ..